homeassistant/components/hunterdouglas_powerview/number.py

```diff
@@ -0,0 +1,125 @@
+"""Support for hunterdouglass_powerview sensors."""
```

Comment:
Let's remove the number platform from this PR. I'd like to merge this version back into `dev` before adding new platforms

Reply:
i thought the whole point of the 'powerview_3_integration' was to get all features added then merge the feature branch into dev ?

Why would i not just target the dev branch for these PR's otherwiese ?

Reply:
With the turns and merge issues, I think we may have lost sight of their original goal (or I did a poor job explaining it in the first place) 

I was working under the premise that the goal was merge in small digestible pieces into this branch, test everything on the old model, and then start adding the v3 new platforms after it's merged to dev.

Reply:
If it would help to do a call to align on this, feel free to reach out on discord

Reply:
I think the only disconnect here was that the initial PR had to be large, because of the degree of change in the upstream API.

Then i went and made a royal mess of the git history.
If you take a look at the individual commits I made you will see there wasnt much changed here.

The last PR was to bring the base functionality, this ones focus was to bring the number platform and should have only changed 3 files. But we also fixed the scene tests here

I'll strip the number platform shortly, and then once merged to dev i will resubmit the PR etc